main.py

```diff
@@ -1 +1,24 @@
 #Mad Libs Generator Project
+
+#Loop back to this point once code finishes
+loop = 1
```

Comment:
```suggestion
loop = 0
```
Maybe try starting the loop at 0 or increasing the range to < 11 in while loop